slow_query_test.go

```diff
@@ -7,14 +7,14 @@ import (
 )
 
 func TestParsingDerivedFromValue(t *testing.T) {
-	value := `SELECT * FROM abacus101_shard6.transactions WHERE balance = 13.37`
+	value := `SELECT * FROM abacus101_shard6.transactions WHERE balance = '13.37'`
```

Comment:
Should we leave this as `= 13.37` to show that floats are no longer filteed?

Reply:
There aren't filtered even as strings. How will the queries actually look coming into timber?